site2/docs/helm-prepare.md

```diff
@@ -4,34 +4,44 @@ title: Prepare Kubernetes resources
 sidebar_label: "Prepare"
 ---
 
-For a fully functional Pulsar cluster, you need a few resources before deploying the Apache Pulsar Helm chart. The following provides instructions to prepare the Kubernetes cluster before deploying the Pulsar Helm chart.
+For a fully functional Pulsar cluster, you need a few resources before deploying the Apache Pulsar Helm chart. This guide provides instructions to prepare the Kubernetes cluster before deploying the Pulsar Helm chart.
 
-- [Google Kubernetes Engine](#google-kubernetes-engine)
-  - [Manual cluster creation](#manual-cluster-creation)
-  - [Scripted cluster creation](#scripted-cluster-creation)
-  - [Create cluster with local SSDs](#create-cluster-with-local-ssds)
+## Prerequisites
 
-## Google Kubernetes Engine
+Set up your environment by installing the required tools.
 
-To get started easier, a script is provided to create the cluster automatically. Alternatively, a cluster can be created manually as well.
+### Install kubectl
 
-### Manual cluster creation
+`kubectl` 1.18 or higher is the required tool that talks to the Kubernetes API. It needs to be compatible with your cluster ([+/- 1 minor release from your cluster](https://kubernetes.io/docs/tasks/tools/install-kubectl/#before-you-begin)). The server version of `kubectl` cannot be obtained until you connect to a cluster.
+
+For the installation instructions, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl).
+
+### Install Helm
+
+Helm is the package manager for Kubernetes. The Apache Pulsar Helm Chart is supported with Helm v3 (3.0.2 or higher).
+
+You can get the Helm from the project's [releases page](https://github.com/helm/helm/releases), or follow other options under the official documentation of [installing Helm](https://helm.sh/docs/intro/install/).
+
+
+## Create Kubernetes cluster
+
+A Kubernetes cluster version 1.18 or higher is required for continuing the deployment. To create a cluster, you can either use a script to automate the process or manually do it.
+
+### Create a cluster manually 
 
 To provision a Kubernetes cluster manually, follow the [GKE instructions](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster).
 
-### Scripted cluster creation
+### Create a cluster using a bootstrap script
```

Comment:
This part should be removed also. There is no need for the `apache/pulsar` document to introduce the steps of how to provision the Kubernetes cluster.